iceoryx_binding_c/test/moduletests/test_publisher.cpp

```diff
@@ -158,24 +158,69 @@ TEST_F(iox_pub_test, noSubscribersAfterUnsubscribe)
     EXPECT_FALSE(iox_pub_has_subscribers(&m_sut));
 }
 
+TEST_F(iox_pub_test, configureCustomHeaderIsSuccessful)
```

Comment:
Capitalise all unit test cases?

Reply:
I would say that it's not part of this PR :)

Reply:
I do like consistency but please prioritize the important things. This is not one of them, besides I prefer the lower case version.